README.md

```diff
@@ -1,9 +1,111 @@
-# Replication
+# Replication [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=replication&metric=alert_status)](https://sonarcloud.io/dashboard?id=replication) [![Known Vulnerabilities](https://snyk.io/test/github/connexta/replication/badge.svg)](https://snyk.io/test/github/connexta/replication)
 
+## Overview
+Replication is the process of creating a copy of a subset of data and storing it on another DDF or HDFS based system. Data can be pulled from a remote DDF and saved to another DDF or HDFS system. Metacards produced by replication are marked with a "Replication Origins" attribute and a tag of "replicated". Replication will automatically start transferring data once an admin creates a replication configuration.
```

Comment:
First sentence - `...another DDF- or HDFS-based system.`

Third sentence - the period should be inside the quotes around `replicated`- i.e., `...tag of "replicated."`